.github/pull_request_template.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Thank you for contributing to TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+-->
+
+## What have you changed? (mandatory)
+
+<!--
+Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
+- Summarize your change (mandatory)
+- How does this PR work? Need a brief introduction for the changed logic (optional)
+- Separately describe each logical change and avoid lazy messages (optional)
+- Describe any limitations of the current code (optional)
+
+Please **NOTE** that:
+- Do not assume reviewers understand the original issue
+-->
+
+## What is the type of the changes? (mandatory)
+
+<!--
+The currently defined types are listed below, please pick one of the types for this PR by removing the others.
+-->
+
+- New feature (non-breaking change which adds functionality)
+- Improvement (non-breaking change which is an improvement to an existing feature)
+- Bug fix (non-breaking change which fixes an issue)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
Just come into my mind: is it better we have a "cherry-pick" type ?

Reply:
You could find it on the label section.